infra/modules/aws-psoxy-output-bucket/main.tf

```diff
@@ -0,0 +1,109 @@
+
+resource "aws_s3_bucket" "output" {
+  # note: this ends up with a long UTC time-stamp + random number appended to it to form the bucket name
+  bucket_prefix = "psoxy-${var.instance_id}-"
```

Comment:
could we use the customer id too (if available)? from out point of view all these buckets look similar.
In any case, with the new export connectors to come should not be an issue anymore

Reply:
So this is the S3 bucket that we import customer's sanitized data from. we shouldn't be touching it directly; they should copy-paste the value into the "Add Connection" flow.

Security on this is that within our platform, only the specific customer's tenant SA can access it. So even if we did for some reason configure the connection on behalf of the customer, and mixed up buckets across customer, should be 403 from AWS IAM when the tenant SA tries to assume a role which it can't assume (based on the role's config in customer AWS)